src/components/WalletAssetsHeadline.vue

```diff
@@ -85,6 +85,10 @@ export default class WalletAssetsHeadline extends Mixins(TranslationMixin, Loadi
     this.updateFilters(Filter.option, value);
   }
 
+  getLabel(index: number): string {
```

Comment:
should it be translated?

Reply:
no, label has internal value only needed to switch between available options. Translations are in another place